internal/client/noop.go

```diff
@@ -21,6 +21,6 @@ func (c *App) Noop(ctx context.Context) error {
 	}
 
 	// Execute it
-	_, err := c.doJob(ctx, job)
+	_, err := c.doJob(ctx, job, nil)
```

Comment:
This is probably wrong, not compiling?

Reply:
Oh strange! I fixed those but forgot to push the files. Should be in now.